examples/src/main/java/com/ibm/watson/discovery/v2/DiscoveryV2Example.java

```diff
@@ -0,0 +1,64 @@
+import com.ibm.cloud.sdk.core.security.Authenticator;
+import com.ibm.cloud.sdk.core.security.BearerTokenAuthenticator;
+import com.ibm.watson.discovery.v2.Discovery;
+import com.ibm.watson.discovery.v2.model.AddDocumentOptions;
+import com.ibm.watson.discovery.v2.model.DeleteDocumentOptions;
+import com.ibm.watson.discovery.v2.model.DocumentAccepted;
+import com.ibm.watson.discovery.v2.model.QueryOptions;
+import com.ibm.watson.discovery.v2.model.QueryResponse;
+import com.ibm.watson.discovery.v2.model.QueryResult;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class DiscoveryV2Example {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/app/watson-developer-cloud/java-sdk/pullRequest?prid=4552601)